FlorenceBML/BMLor403/BMLor403.xml

```diff
@@ -34,8 +35,421 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    ሃይማኖት፡ ውስተ፡ ኣሚነ፡ ዚአሁ፡ ወቅዱስ፡ መንፈስ፡ ወይጸግወነ፡ ሣህሎ፡ ወምሕረቶ፡ ወይስረይ፡ ኃጢአተነ፡ በብዙኃ፡ ምሕረቱ፡ ስብሓት፡ ለኪ፡ ኦወላዲተ፡ እግዚአ፡ ኵሉ፡ 
+                                    አኰቴት፡ ወክብር፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ይአዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም። ኣሜን። ወኣሜን።</explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i6">
+                                <locus from="160v" to="164r"/>
+                                <title ref="LIT3138Negeranni"/>
+                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ጸሎት፡ ዘተስእለቶ፡ ማርያም፡ ለእግዚእትነ፡ ወትቤሎ፡ ንግረ፡ ስምከ፡ ወይቤላ። 
+                                    እነግረኪ፡ ስምየ፡ ዘዕፁብ፡ ወግሩም፡ ዘስምዮቱ፡ ለስምየ። ወትቤሎን፡ ንግረኒ፡ ስምከ። </incipit>
+                                <explicit xml:lang="gez"><gap reason="illegible"/>ነፍስየ፡ ወሥጋየ፡ ወውሉድየ፡ ወንዋይየ፡ ወጥሪትየ፡ ዘቤት፡ ወዘገዳም፡ ዘርኁቅ፡ ወዘቅሩብ፡ በእደ፡ 
+                                    እግዚአብሔር፡ የማናይ። በዘ፡ አቀማ፡ ወአኀዛ፡ ለዓለም፡ በዘ፡ አነሞ፡ ሰሰማይ፡ በአጽባዕቱ። በተባዕታይ፡ ማይ። እንዘ፡ ስፍሑ፡ አብ፡ ወፋእሙኒ፡ ወልድ። ወመንፈስ፡ ቅዱስ፡
+                                    መስቄሁ፡ ወፍጻሜሁ፡ በዝንቱ፡ አስማት፡ ሥሉስ፡ ተማኅፀንኩ፡ አነ፡ ገብርከ። <gap reason="illegible"/>ለዓለመ፡ ዓለም። አሜን። አሜን። ወአሜን። ለይኩን። 
+                                    ለይኩን። ለይኩን።</explicit>
+                                <note>The name of the owner <ref target="PRS0000Simon">ስምዖን</ref> was inscribed on <locus target="#163v #164r"/>, 
+                                    but has been erased.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i7">
+                                <locus from="165" to="171r"/>
+                                <title ref="LIT4299Prayer"/>
+                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ጸሎት፡ ዘእግዝእትነ፡ ማርያያም። ኢየሱስ፡ ክርስቶስ። ትፍሥሕት፡ ለኵሉ፡ 
+                                    ዓለመ። ኢየሱስ፡ ክርስቶስ፡ ያስተፈሥሕ፡ ኵሉ፡ ልብ። ኢየሱስ፡ ክርስቶስ፡ አንተ፡ ብርሃን፡ ለዕውራን። ኢየሱስ፡ ክርስቶስ፡ አንተ፡ ልብስ፡ ለዕሩቃን።</incipit>
+                                <explicit xml:lang="gez">ወትዕቀበኒ፡ ወትዕቀበኒ፡ ወትትማኅፀነኒ፡ ወትባልሐኒ፡ ወተሣሀለኒ፡ በኵሉ፡ መንባርትየ፡ ወበኵሉ፡ ፍኖትየ፡ ወበኵሉ፡ መንግድየ። ኀበ፡ አኀውር፡ 
+                                    ወኀበ፡ እወፅእ፡ ወኀበ፡ እገብእ። ኀበ፡ እወፍር። ወኀበ፡ አአቱ፡ ወኀበ፡ እቀውም። ወኀበ፡ እትቀመጥ። ወትመርሐኒ፡ ወትስረይ፡ ሊተ፡ ኵሉ፡ ኃጢአትየ፡ በዳግም፡ ምጽአትከ። ኢየ። 
+                                    ኢየ። ኢየ። ዳኬ። ዳኬ። ዳኬ። ለዓለመ፡ ዓለም፡ አሜን። አሜን። ወአሜን።</explicit>
+                                <note>The name of the owner <ref target="PRS0000Simon">ስምዖን</ref> was inscribed on <locus target="#167r #168r #168v"/>, 
+                                    but has been erased.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i8">
+                                <locus from="171r" to="174v"/>
+                                <title>Prayer invoking the sufferings of Christ</title>
+                                <incipit xml:lang="gez">እምድንግል፡ ተወሊዶ፡ ከመ፡ ፈቃደከ፡ ይፈጽም፡ ወሕዝብ፡ ለከ፡ ይሥራዕ፡ ቅዱስ። ሰፍሐ፡ እደዊሁ፡ ለሕማም፡ ሐሚሞ፡ ከመ፡ ለሕሙማን፡ ይፍታሕ፡ 
```

Comment:
The incipit seems to be identical with text contained on BMLsmarco741, folio 63/64 (as they are numbered in the Transkribus text, I didn't check the manuscript now). Maybe it could be worthwhile to check whether the correspondence continues and the text can be identified with an anaphora?

Reply:
Me also I found a stunning similarities when looking up the incipit in BM-website (looking for ሰፍሐ in simple search). But I was not able to identify the passage that is given in BM in the manuscript images of folio 63 or 64 (also not in the scan-no 63 and 64). I do not know, what the numbers 63 and 64 may refer to, but folio 63 and 64 is definitely incorrect.

Reply:
Infact the combination of ሰፍሐ፡ እደዊሁ፡ and ሕሙማን and ሕዝብ፡ is not that much extraordinary when it comes to the crucification of Jesus Christ. So the correlation maybe accidental anyways.

Reply:
In the scan image 70, I find folio 61v and 62r. In scan image 71, I see 62v and 63r. None of these contain the aforementioned words.

Reply:
I do see these words in scan image 71, f. 62v

Reply:
I don't. I am sorry!

Reply:
I am sorry! I was mistaken and looked up the wrong manuscript. I see can see the words mentioned before and I will check the text for more similarities in a minute. It is folio 66v on image 71, by the way.

Reply:
It is infact 67r, numbered in the manuscript as 66r.

Reply:
I have gone through all the images and transcribus here https://github.com/BetaMasaheft/Manuscripts/pull/2059 and I am pretty sure that the pencil foliation is correct, image 71 is 66v-67r. 
There was one image photographed twice (image 55 and 56 both are on ff. 51v-52r). Anaphora in 741 begins, as foliated, on 65r (image 69).

Reply:
I read through one third of this textual unit and I can confirm, that it is indeed the same text. Unless it shows a lot of deviations, I think it is reasonable to define this passage as LIT1104Anapho.

Reply:
> I have gone through all the images and transcribus here #2059 and I am pretty sure that the pencil foliation is correct, image 71 is 66v-67r. There was one image photographed twice (image 55 and 56 both are on ff. 51v-52r). Anaphora in 741 begins, as foliated, on 65r (image 69).

OK. Shall I open a new branch to correct the loci in all the msItems in BMLsmarco741?

Reply:
I have, and you have already approved the Pull Request, have you not? https://github.com/BetaMasaheft/Manuscripts/pull/2059

Reply:
Yes, but all the other loci in the physDesc and in the Contents need to be revised now, or not? Did you check them all, already?

Reply:
No, sorry, I only changed for the msItems. Go ahead then, thank you

Reply:
I checked the loci in the quire record and in the entire physDesc. It is everything alright now.